unit-tests/live/rec-play/playback_helper.py

```diff
@@ -0,0 +1,53 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+
+import time
+from rspy import log, test
+from rspy.timer import Timer
+
+'''
+This class is used to wait and verify a requested playback status arrived whithin a requested timeout.
+It blocked the calling thread and sample the playback status using a playback status callback from LRS API\
```

Comment:
`\` a the end

Reply:
`blocked` -> `blocks`
`sample` -> `samples`

Reply:
👍